libs/components/forms/src/lib/modules/checkbox/checkbox.component.ts

```diff
@@ -162,55 +173,63 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
    * @default false
    */
   @Input()
-  set required(value: boolean) {
-    this._required = SkyFormsUtility.coerceBooleanProperty(value);
+  set required(value: boolean | undefined) {
+    this.#_required = SkyFormsUtility.coerceBooleanProperty(value);
   }
 
   get required(): boolean {
-    return this._required;
+    return this.#_required;
   }
 
   /**
    * Fires when users select or deselect the checkbox.
    */
   @Output()
   public get checkedChange(): Observable<boolean> {
-    return this._checkedChange;
+    return this.#_checkedChange;
```

Comment:
We need to use the `asObservable` pattern for these.
See: https://github.com/blackbaud/skyux/blob/main/libs/components/core/src/lib/modules/overlay/overlay.component.ts#L58-L60